Sources/Fullscreen/EasterEggs/DrumMachine/Demo/DrumMachineDemoView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+public class DrumMachineDemoView: UIView {
+    private lazy var drumMachineView: DrumMachineView = {
+        let drumMachineView = DrumMachineView()
+        drumMachineView.translatesAutoresizingMaskIntoConstraints = false
+        return drumMachineView
+    }()
+
+    // MARK: - Init
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Lifecycle
+
+    public override func didMoveToSuperview() {
+        super.didMoveToSuperview()
+        if superview == nil {
+            drumMachineView.stop()
+        } else {
+            drumMachineView.start()
+        }
+    }
+
+    // MARK: - Setup
+
+    private func setup() {
+        addSubview(drumMachineView)
+
+        NSLayoutConstraint.activate([
```

Comment:
You could use `drumMachineView.fillInSuperview()` instead of this. We didn't add more AutoLayout convenience methods but for sure we're open for suggestions :)

Reply:
I think it's the most useful one 😉